src/squid.md

```diff
@@ -0,0 +1,247 @@
+---
+title: squid
+section: 8
+---
+# NAME
+
+**squid** - HTTP web proxy caching server
+
+# SYNOPSIS
+
+**squid** [-dsCFNRSXY] [-\-foreground] [-l facility] [-f file] [-[au] port] [-k signal] [-n service]
+
+**squid** -[hv]
+
+**squid** -n service -i [-dsX] [-\-foreground] [-l facility]
+
+**squid** -n service -r [-dsX] [-\-foreground] [-l facility]
+
+**squid** -n service -O command [-ds] [-\-foreground] [-l facility]
+
+**squid** -z [-NS] [-dsX] [-\-foreground] [-l facility] [-f file] [-n service]
```

Comment:
This PR does not have to improve the old broken synopsis, but if we are going to do that, let's be consistent about listing options that require an argument. Option `-d` requires an argument but is shown as if it does not.